src/controllers/ContinuedIllnessController.ts

```diff
@@ -0,0 +1,77 @@
+import Joi from '@hapi/joi';
+import { SessionMiddleware } from 'ch-node-session-handler';
+import { controller } from 'inversify-express-utils';
+import { BaseController } from './BaseController';
+import { FormValidator } from './validators/FormValidator';
+
+import { AuthMiddleware } from 'app/middleware/AuthMiddleware';
+import { IllnessReasonFeatureMiddleware } from 'app/middleware/IllnessReasonFeatureMiddleware';
+import { loggerInstance } from 'app/middleware/Logger';
+import { Appeal } from 'app/models/Appeal';
+import { Illness } from 'app/models/Illness';
+import { YesNo } from 'app/models/fields/YesNo';
+import { createSchema } from 'app/models/fields/YesNo.schema';
+import { CONTINUED_ILLNESS_PAGE_URI } from 'app/utils/Paths';
+import { Navigation } from 'app/utils/navigation/navigation';
+
+const template = 'continued-illness';
+
+const navigation : Navigation = {
+    previous(): string {
+        return '';
+    },
+    next(): string {
+        return '';
+    },
+    actions: (_: boolean) => {
+        return {
+            continue:'action=continue'
+        };
+    }
+};
+
+@controller(CONTINUED_ILLNESS_PAGE_URI, IllnessReasonFeatureMiddleware, SessionMiddleware, AuthMiddleware)
+export class ContinuedIllnessController extends BaseController<any> {
+
+    constructor() {
+        const errorMessage = 'You must tell us if this is a continued illness';
+        const schema: Joi.AnySchema = Joi.object({
+            continuedIllness: createSchema(errorMessage)
+        }).unknown(true);
+
+        super(
+            template,
+            navigation,
+            new FormValidator(schema)
+        );
+    }
+
+    protected prepareViewModelFromAppeal(appeal: Appeal): any {
+        const illnessStartDateRaw = appeal.reasons.illness?.illnessStart;
+
+        if (!illnessStartDateRaw) {
+            throw new Error('Illness Start Date expected but not found');
+        } else {
+            const illnessStart = illnessStartDateRaw.toLocaleDateString('en-GB', {
+                year: 'numeric', month: 'long', day: 'numeric'
+            });
+            const hint: string = `You told us the illness started on ${illnessStart}`;
+            return { hint };
```

Comment:
I'd like to suggest different solution.

I prefer to pass pure data to the template (start date in this instance) and then format it accordingly if needed instead of building UI text in the controllers.

I suggest creating custom Nunjucks filter for date formatting and then using that filter in template as `{{ startDate | formatDate('[format here]') }}`.

This way:
1. wording of the UI remains in template
2. `formatDate` can be reused in many places
3. `formatDate` can be unit tested

Reply:
Do we have any instances of custom Nunjucks filters elsewhere in the code? I did this to avoid meddling with the BaseController but can of course look into this. I originally wanted to only pass the formatted date to the template, but Nunjucks doesn't support nested variables (this occurs within a `govukRadios` macro).

Reply:
Nope, but here is an example from project I previously worked for: https://github.com/hmcts/cmc-citizen-frontend/blob/master/src/main/modules/nunjucks/filters/dateFilter.ts#L18

Reply:
Nunjucks filters have no dependency on BaseController.

In nutshell you would need to:
1. implement filter method
2. register filter with templating engine
3. use registered filter in the code

If we have one date format you can make it simple / argument-less for now so it always formats date into one format.

Reply:
Okay, thanks for the explanation. I'll get on that - though I'll get in touch with Natalie to see what datestring format she's using first.